com.gitlab.stonekingdoms.StoneKingdoms.json

```diff
@@ -0,0 +1,95 @@
+{
+    "app-id": "com.gitlab.stonekingdoms.StoneKingdoms",
```

Comment:
it should be `io.gitlab`.

https://docs.flatpak.org/en/latest/conventions.html?highlight=app-id#application-ids

Reply:
also it should be `stone_kingdoms`. `-` are replaced with `_`. Same link.